From: Bastien Nocera <hadess@hadess.net>
Date: Tue, 26 Oct 2021 11:57:30 +0200
Subject: hwdb: Allow console users access to media* nodes

Newer webcams and video devices have controls only available through
/dev/media* nodes. Make sure they're accessible in the same way as
/dev/video* nodes.

Closes: #21054
(cherry picked from commit 63fbfc598f9f6d3ce34cc00a7687089dab24daff)
---
 src/login/70-uaccess.rules.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/login/70-uaccess.rules.in b/src/login/70-uaccess.rules.in
index 56e1087..1b6be82 100644
--- a/src/login/70-uaccess.rules.in
+++ b/src/login/70-uaccess.rules.in
@@ -33,6 +33,7 @@ SUBSYSTEM=="sound", TAG+="uaccess", \
 # Webcams, frame grabber, TV cards
 SUBSYSTEM=="video4linux", TAG+="uaccess"
 SUBSYSTEM=="dvb", TAG+="uaccess"
+SUBSYSTEM=="media", TAG+="uaccess"
 
 # industrial cameras, some webcams, camcorders, set-top boxes, TV sets, audio devices, and more
 SUBSYSTEM=="firewire", TEST=="units", ENV{IEEE1394_UNIT_FUNCTION_MIDI}=="1", TAG+="uaccess"
